docs/docs/aleph_zero/overview.md

```diff
@@ -57,9 +56,9 @@ The "Logic" folder hosts a suite of mathematical calculations which are primaril
 
 The "Test Helpers" directory contains macros designed for efficient end-to-end testing. These macros abstract low-level calls and transaction building, allowing developers to focus solely on verifying expected logic during tests. This minimizes code repetition, simplifies the testing interface, and ensures a clear and concise testing environment.
 
-### e2e
+### E2E
```

Comment:
Both 'e2e' and 'E2E' are correct. IMO 'e2e' would be better, as it directly reflects the code 1:1